org-cyf-itd/content/blocks/step-6/instructions/index.md

```diff
@@ -1,36 +1,55 @@
 +++
 title ="Step 6: What to do"
-description= "Take the Duolingo Practice Test"
+description= "Use AI appropriately to solve a noval CSS problem"
 emoji= "🤖"
-time= 60
+time= 90
 [build]
   render = 'never'
   list = 'local'
   publishResources = false 
 +++
 
 ### Overview
+As covered in step 2, generative AI tools such as ChatGPT can be a great help in your studies with CodeYourFuture. However, the way you use them is very important. These tools can just as easily be a great obstacle to your learning if you let them do your thinking for you.
 
-Participants are required to score 100 or higher on the Duolingo English Practice Test, which corresponds to CEFR level B2. We set this level because we have learned from experience that participants without it are unable to follow the course materials in subsequent courses.
+This step is an opportunity to practice this with a practical task which is outside your current knowledge of CSS and HTML. We have provided a [CodePen](https://codepen.io/Ara225/pen/JoYbRVd) which contains some HTML code. Your task is to use CSS to style the HTML so that:
+- The element with the class of "parent" has a dotted border
+- The element with the class of "parent" is centered on the page even if the screen size changes
+- The div elements with the class "child" have background colors, borders, rounded corners and margins between them.
+- The div elements with the class "child" stay inside the element with the class parent even if the screen size changes
 
-**We want you to succeed.** If you have not yet reached this level, it is better to work on your English _first_, then apply to us when you are _ready_.
+You must not change the HTML in anyway.
 
+You may use any colors you like - they aren't the important bit in this task.
 
-### What Should You Do?
+Here are some visual examples:
+
+#### On a large screen
+![alt text](animals-larger-screen.png)
+#### On a much smaller screen
+![alt text](animals-smaller-screen.png)
 
-##### 1. Preparation
+Assuming you were new to HTML and CSS when you started this course, this task is outside your current knowledge. This makes it an excellent candidate for AI assistance, as you can use AI to help you understand the new concepts.
+
+### What Should You Do?
 
-1. Find 50 minutes where you will not be interrupted.
-2. [Register on the Duolingo website](https://englishtest.duolingo.com/register) to do the Practice Test.
-3. You need to look at your camera during some exercises, so test that your camera and microphone are working. You may need to do this test on your phone.
+#### 1 - Fork the CodePen
+Fork the [exercise CodePen](https://codepen.io/Ara225/pen/JoYbRVd) into your CodePen account
```

Comment:
Do we know they have an account at this stage? If not, we should tell them to create one if needed.

Reply:
Will edit to remind them, probably sensible